src/choose-location/choose-location.component.tsx

```diff
@@ -6,16 +6,19 @@ import { navigate, useConfig } from '@openmrs/esm-framework';
 import { setSessionLocation, queryLocations } from './choose-location.resource';
 import { useCurrentUser } from '../CurrentUserContext';
 import { LocationEntry } from '../types';
+import { StaticContext } from 'react-router';
```

Comment:
I think this is rather used as a type, but as `react-router-dom` is the only package shared here we should also use `react-router-dom` (or is this type not exposed there?).

Reply:
Not sure, I just relied on IntelliSense - but I'll have a look!

Reply:
It's not actually exported by react-router-dom.
I only added the type because TS would be complaining otherwise, i.e. it has no real purpose. I guess we could also go with `unknown`/`any` as an alternative if TS accepts it.

Reply:
Feared that. Well, just to be sure the type gets erased as it should - make it an `import type { ... } from ...` to make sure / communicate that this is only a type import. Thanks!

Reply:
TIL that `import type` is possible.

Reply:
Should be fine now.